### ISSUE TITLE
Too long since last issue, had to do something

### ISSUE THREAD

**Issue by adelphin** [jpromerob/HL2019_Project#160]
Hello Francesco

**Comment by jpromerob** [comment]
@adelphin, what do you want us to do?

**Comment by adelphin** [comment]
I believe Pr. F. Basso should do more stuff

**Comment by jpromerob** [comment]
I agree ... Please @cescob ... wake up and do more stuff!!!

**Comment by cescob** [comment]
I saw it yesterday :p but I decided to wait to answer here!!! It would have been too easy to send you a wa txt

**Comment by cescob** [comment]
And I'm still doing.........something. I'm sure @adelphin you will be amazed by some fancy colourful draw that I'm doing on Matlab ;-)

**Comment by adelphin** [comment]
Yes please give us some figures!!! Sorry your mail was drown under those I receive from my lab :o 
Here are some colorful stuff from my current work ;)
<img width="1820" alt="modeldynmrfbssfp" src="https://user-images.githubusercontent.com/35633910/47439741-339c5e80-d7ad-11e8-9bc1-3b9a52ea9d2b.png">

**Comment by jpromerob** [comment]
@adelphin ... Wow ... the figures on the left plus a bit of Pink Floyd / Jimmy Hendrix and we're back to the 60's/70's

**Comment by cescob** [comment]
I would say they are more like modern pop art. 
I have these fancy candy 
![cattura53546](https://user-images.githubusercontent.com/35661467/47495316-18385e80-d854-11e8-8deb-03e059e2781d.JPG)

**Comment by adelphin** [comment]
Yummy and strangely satisfying to watch :o
